### DISCUSSION TITLE
Documentation Page for Vue Filter Service is not reachable from either navigation or search

### DISCUSSION THREAD

**Discussion by santiago** [primefaces/community#786] [PrimeVue]
Trying to find the documentation for DataTable filtering I had a hard time trying to get to the FilterService page. Even searching for `FilterService` gives no results. I could fortunately get back to it from my bookmarks.

**Comment by tugcekucukoglu** [comment]
We removed it from the documentation. We can do something different for this.

Sorry for the confusion.
